cmd/crc/cmd/stop.go

```diff
@@ -42,11 +42,7 @@ func runStop(arguments []string) {
 			// Most of the time force kill don't work and libvirt throw
 			// Device or resource busy error. To make sure we give some
 			// graceful time to cluster before kill it.
-			if globalForce {
-				killVM(killConfig)
-				errors.Exit(0)
-			}
-			yes := input.PromptUserForYesOrNo("Do you want to force power off")
+			yes := input.PromptUserForYesOrNo("Do you want to force power off", globalForce)
```

Comment:
Might have been more natural to have a first commit introducing globalForce, second one introduces PromptUserForYesOrNo(), and 3rd one adds clear-cache.

Reply:
Okay, that makes sense, let me reorder those!

Reply:
Fixed up the commits, have a look at your leisure!